features/bootstrap/FeatureContext.php

```diff
@@ -569,6 +569,9 @@ public function thereIsAFileConfigDummyObject()
     public function thereIsAFooEntityWithRelatedBars()
     {
         $foo = new DummyCar();
+        $foo->setName('mustli');
```

Comment:
A car called mustli ?

Reply:
so?

Reply:
Your nickname is Simperfit, so why not Mustli for a car? 😄